js/main.js

```diff
@@ -59,3 +59,19 @@ $webPage.addEventListener('keydown', function (event) {
     $currentWord.className = 'word active';
   }
 });
+
+$webPage.addEventListener('keydown', function (event) {
+  if (event.key === 'Tab') {
+    $p.textContent = '';
+    currentCharacter = 0;
+    gameLoading();
+  }
+});
+
+window.addEventListener('keydown', function (event) {
+  if (event.key === ' ' && event.target === document.body) {
+    event.preventDefault();
+  } else if (event.key === 'Tab' && event.target === document.body) {
+    event.preventDefault();
+  }
+});
```

Comment:
What is the purpose of this? It may be fine, but I'm not fully clear on what it is doing.